src/Psl/Filesystem/exists.php

```diff
@@ -9,9 +9,7 @@
 /**
  * Check whether $node exists.
  *
- * @param string $node Path, absolute or relative to the current working directory.
- *
- * @psalm-assert-if-true non-empty-string $node
```

Comment:
The assert here causes NoValue errors all over the place. I think it's better to make it non-empty to start with

Reply:
can you open a bug for psalm about this? if able to replicate it.

Reply:
It has been reported already: https://github.com/vimeo/psalm/issues/8659